R/osmactive.R

```diff
@@ -818,7 +818,12 @@ estimate_traffic = function(osm) {
         highway == "tertiary_link" ~ 3000,
         highway == "residential" ~ 500,
         highway == "service" ~ 500,
-        highway == "unclassified" ~ 500
+        highway == "unclassified" ~ 500,
+        highway == "living" ~ 500,
```

Comment:
You could replace this with either 

```
        highway == "living_street" ~ 500,
```

Or perhaps better

```suggestion
        str_detect(highway, pattern = "living") ~ 500,
```